src/context/walletContext/zeroDevContext.context.tsx

```
@@ -0,0 +1,328 @@
+    isSendingUserOp: boolean
+    setIsSendingUserOp: (sendingUserOp: boolean) => void
+    handleRegister: (username: string) => Promise<void>
+    handleLogin: (username: string) => Promise<void>
+    handleSendUserOpEncoded: (
+        {
+            to,
+            value,
+            data
+        }: {
+            to: string,
+            value: BigInt | undefined,
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)